.gitignore

```diff
@@ -40,8 +40,6 @@ Vagrantfile
 /keys/*.key.enc
 !/keys/*.key.enc.example
 /keys/equifax_rsa
-/keys/equifax_rsa.pub
```

Comment:
I think we don't want to ignore any of these, not just the one. When I run git status on your branch, this is what I see:

![image](https://user-images.githubusercontent.com/963654/44163206-5cef8e80-a088-11e8-9371-d443f2518348.png)

Everyone who's been working on this since before we stoped generating those on `make setup` will see the same thing. We'll want to leave these in so they don't accidentally get checked in by one of us.

Reply:
Done.